docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -131,29 +147,29 @@ ID        Time                 Host        Tags        Paths
 1 snapshots
 {{< /output >}}
 
-### Run the backup script automatically
+## Set Up Automated Database Backups
 
 Linux has several ways of running a job on a schedule basis. Pick one of the methods below to configure the backup script to run periodically.
 
 {{< note >}}
 Consider your databases' usage, how much data you could potentially lose, and the storage space required when choosing how often to run your script.
 {{< /note >}}
 
-### 1. Cron
+### Cron
 
 Edit your "crontab" file:
 
     sudo crontab -e
 
-Add a line for your backup script. This example runs the backup every hour, on the hour. See the [Schedule tasks with Cron](https://www.linode.com/docs/tools-reference/tools/schedule-tasks-with-cron/) article for additional scheduling options.
+Add a line for your backup script. This example runs the backup every hour, on the hour. See the [Schedule tasks with Cron](/docs/tools-reference/tools/schedule-tasks-with-cron/) article for additional scheduling options.
 
     0 * * * * /usr/local/bin/backup_mariadb > /tmp/mariadb-backup-log.txt 2>&1
 
-### 2. Systemd
+### Systemd
 
 Systemd can run commands (known as "units") on a periodic basis using "timers". You can use systemd commands to monitor when the timers and commands last ran, and the output from them.
 
-To schedule a command, you need 2 configuration files - the command to run (known as the service) and when to run it (the timer.)
+To schedule a command, you need 2 configuration files: the command to run (known as the service) and when to run it (the timer).
```

Comment:
This might just be my own peculiarities with the punctuation. Not really a big deal, though